### DISCUSSION TITLE
Change default SSID of AI-deck

### DISCUSSION THREAD

**Discussion by scf-ku** [bitcraze/discussions#1579] [Q&A]
I am also trying to work crazyflie with AI-deck.

I have already attached AI-deck and I could find an access point, SSID: "Bitcraze AI-deck example".

Is there any way to change this SSID with each crazyflies?

Best Regards.

**Comment by knmcguire** [comment]
Hi! Have you gone through the [getting started tutorial of the AIdeck](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/)? if so, could you point to which part of the tutorial you get stuck.

In the tutorial it states that you should see `WiFi streaming example`, not `Bitcraze AI-deck example`. 

After the tutorial is done, there are some resources for the next steps, that includes a link to insturctions to change the ssid.
